util/kb.c

```diff
@@ -393,7 +393,11 @@ redis_new (kb_t *kb, const char *kb_path)
 
   kbr = g_malloc0 (sizeof (struct kb_redis) + strlen (kb_path) + 1);
   kbr->kb.kb_ops = &KBRedisOperations;
-  strncpy (kbr->path, kb_path, strlen (kb_path));
+  #pragma GCC diagnostic push
+  #pragma GCC diagnostic ignored "-Wstringop-truncation"
+  #pragma GCC diagnostic ignored "-Wstringop-overflow"
+    strncpy (kbr->path, kb_path, strlen (kb_path));
```

Comment:
I guess the more correct way would be fixing it, instead of ingoring.
Afaik the problem is, that `strncpy` uses the last byte to null-terminate the copy  with `\0`.
So the compiler warns, that if the string is really as long as `kb_path` it can't be null-terminated.
The compiler assumes that the given string is not terminated.
I guess we should do something like:
```suggestion
    strncpy (kbr->path, kb_path, strlen (kb_path) - 1);
```

Reply:
I have updated the PR, so the stringop-truncation statement can be removed,